sdk/src/instruction_processor_utils.rs

```diff
@@ -43,6 +42,15 @@ pub fn next_keyed_account<I: Iterator>(iter: &mut I) -> Result<I::Item, Instruct
     iter.next().ok_or(InstructionError::NotEnoughAccountKeys)
 }
 
+/// Return all the signers from a set of KeyedAccounts
+pub fn signers(keyed_accounts: &[KeyedAccount]) -> HashSet<Pubkey> {
```

Comment:
No dependency on `Instruction` here, so you can put this in `account.rs` after `KeyedAccount`.  How about `get_signers` thought, since it's a function, not a method?

Reply:
sounds good to me